cmd/nexusUpload_test.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"github.com/SAP/jenkins-library/pkg/nexus"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
```

Comment:
Test coverage seems to pretty low, we should increase this which likely means some adoptions of the code in `nexusUpload.go`